<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Музыкальный плеер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_progress">
      <item>
       <widget class="QLabel" name="currentTimeLabel">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="progressBar">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalTimeLabel">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_playerControls">
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/prefix/buttons/play.ico</normaloff>:/prefix/buttons/play.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/prefix/buttons/pause.ico</normaloff>:/prefix/buttons/pause.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/prefix/buttons/stop-button.ico</normaloff>:/prefix/buttons/stop-button.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="previousButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/prefix/buttons/back.ico</normaloff>:/prefix/buttons/back.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/prefix/buttons/next.ico</normaloff>:/prefix/buttons/next.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="repeatButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/prefix/buttons/repeat.ico</normaloff>:/prefix/buttons/repeat.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_controls">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="currentTrackLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Нет текущего трека</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_library">
       <attribute name="title">
        <string>Библиотека песен</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1" rowspan="2">
         <widget class="QLabel" name="currentSongListViewTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Библиотека песен</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="albumArtLabel">
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>230</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>230</height>
           </size>
          </property>
          <property name="text">
           <string>No Album Art</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QListView" name="songListView">
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="movement">
           <enum>QListView::Movement::Snap</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="resizeMode">
           <enum>QListView::ResizeMode::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::LayoutMode::Batched</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="deleteSongButton">
          <property name="text">
           <string>Удалить трек</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/prefix/buttons/delete.ico</normaloff>:/prefix/buttons/delete.ico</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="addSongButton">
          <property name="text">
           <string>Добавить трек</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/prefix/buttons/plus.ico</normaloff>:/prefix/buttons/plus.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_playlists">
       <attribute name="title">
        <string>Плейлисты</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_playlists">
        <item>
         <widget class="QListView" name="playlistListView"/>
        </item>
        <item>
         <widget class="QPushButton" name="createPlaylistButton">
          <property name="text">
           <string>Создать плейлист</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePlaylistButton">
          <property name="text">
           <string>Удалить плейлист</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Редактировае метаданных</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="yearLabel">
          <property name="text">
           <string>Год:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="albumLabel">
          <property name="text">
           <string>Альбом:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="editMetadataButton">
          <property name="text">
           <string>Редактировать метаданные</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout_Programm"/>
    <addaction name="actionManual"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout_Programm">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
